c2c/code-to-cloud-spec.md

```diff
@@ -250,21 +249,21 @@ Container image user account related properties
 
 Contains the configs required for the application in terms of Key Value pairs.
 
-|Identifier     |Description	                    |Example Value  |
-|---	        |---	                            |---	        |
-|name   	    |name of the configuration   	    |"foo"   	    |
-|config_name    |Name of the config config map      |"module-foo"   |
-|key   	        |Key of the environment variable    |"FOO"   	    |
+|Identifier     |Description	                                        |Example Value  |
+|---	        |---	                                                |---	        |
+|key_ref   	    |Key of the environment variable                        |"FOO"   	    |
+|name(optional) |Name of the env if its different from the Key          |"foo"   	    | 
+|config_name    |Name of the config config map                          |"module-foo"   |
 
 #### [[cloud.config.secrets]]
 
 Contains external configuration files for the code.
 
-|Identifier   	|Description	                |Example Value   	    |
-|---	        |---	                        |---	                |
-|name   	    |Name of the secret  	        |"ROOT_PASSWORD"  	    |
-|secret_name   	|Name of the secret group       |"db-crdential-secret"  |
-|key_ref   	    |Reference key of the secret    |"MYSQL_ROOT_PASSWORD"  |
+|Identifier   	|Description	                                        |Example Value   	    |
+|---	        |---	                                                |---	                |
+|key_ref   	    |Reference key of the secret                            |"MYSQL_ROOT_PASSWORD"  |
+|name(optional) |Name of the secret if its different from the Key	    |"ROOT_PASSWORD"  	    |
+|secret_name   	|Name of the secret group                               |"db-crdential-secret"  |
```

Comment:
Shall we add a note saying either `key_ref` or `name` should be present and if one is not present then the value from the given field will be set to the other.

Reply:
Resolved in a separate meeting.
Reasoning -  From developers perspective, the key_ref should be specified in the code. It makes more sense to make it mandatory and make name field which represents the config map optional.